script/bulk_import/vbulletin5.rb

```diff
@@ -0,0 +1,779 @@
+# frozen_string_literal: true
+
+require_relative "base"
+require "cgi"
+require "set"
+require "mysql2"
+require "htmlentities"
+require 'ruby-bbcode-to-md'
+require 'find'
+
+class BulkImport::VBulletin5 < BulkImport::Base
+
+  DB_PREFIX = ""
+  SUSPENDED_TILL ||= Date.new(3000, 1, 1)
+  ATTACH_DIR ||= ENV['ATTACH_DIR'] || '/shared/import/data/attachments'
+  AVATAR_DIR ||= ENV['AVATAR_DIR'] || '/shared/import/data/customavatars'
+  ROOT_NODE = 2
+
+  def initialize
+    super
+
+    host     = ENV["DB_HOST"] || "localhost"
+    username = ENV["DB_USERNAME"] || "root"
+    password = ENV["DB_PASSWORD"]
+    database = ENV["DB_NAME"] || "vbulletin"
+    charset  = ENV["DB_CHARSET"] || "utf8"
+
+    @html_entities = HTMLEntities.new
+    @encoding = CHARSET_MAP[charset]
+    @bbcode_to_md = true
+
+    @client = Mysql2::Client.new(
+      host: host,
+      username: username,
+      password: password,
+      database: database,
+      encoding: charset,
+      reconnect: true
+    )
+
+    @client.query_options.merge!(as: :array, cache_rows: false)
+
+    @forum_typeid = mysql_query("SELECT contenttypeid FROM #{DB_PREFIX}contenttype WHERE class='Forum'").to_a[0][0]
```

Comment:
Nitpick, but you could `LIMIT 1` since you're interested in only 1 result

Reply:
It's worth leaving as a note -- I don't have a great way to test the change at this time.